rest/clients/ts/src/client.ts

```diff
@@ -430,13 +430,31 @@ export class PortalSDK {
   public async requestInvoice(
     recipientKey: string,
     content: InvoicePaymentRequestContent) : Promise<InvoiceResponseContent> {
-    
-    const response = await this.sendCommand('RequestInvoice', { recipient_key: recipientKey, content });
+    return this.sendCommand<InvoiceResponseContent>('RequestInvoice', {
+      recipient_key: recipientKey,
+      content
+    });
+  }
 
-    if (response.type === 'invoice_payment') {
-      return response as InvoiceResponseContent ;
-    }
+  /**
+   * Issue a JWT token for a given public key
+   */
+  public async issueJwt(pubkey: string, expiresAt: number): Promise<string> {
+    return this.sendCommand<{ type: 'issue_jwt', token: string }>('IssueJwt', {
+      pubkey,
+      expires_at: expiresAt
+    }).then(response => response.token);
+  }
 
-    throw new Error('Unexpected response type');
+  /**
+   * Verify a JWT token and return the claims
+   */
+  public async verifyJwt(pubkey: string, token: string): Promise<{ pubkey: string}> {
```

Comment:
I can see the CI is failing because you are trying to read `expires_at` from the result of this